code/modules/atmospherics/datum_pipeline.dm

```diff
@@ -113,7 +113,7 @@
 
 	for(var/obj/machinery/atmospherics/pipe/edge in edges)
 		for(var/obj/machinery/atmospherics/result in edge.pipeline_expansion())
-			if(!istype(result,/obj/machinery/atmospherics/pipe) && (result!=reference))
```

Comment:
What was the purpose of this check?

Reply:
Removing this is a somewhat complicated change to pipenet builds; the check was there to try and salvage the situation when the pipe geometry just isn't compatible with having one node per side. It should no longer be needed.